### DISCUSSION TITLE
Getting error on `npx prisma generate`

### DISCUSSION THREAD

**Discussion by Hasilt** [MichalLytek/typegraphql-prisma#86] [Q&A] [answered]
Hello, 

I'm getting this error while running `npx prisma generate`. Can somebody help me to solve this problem?

`Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Error: spawn typegraphql-prisma ENOENT`

**Comment by MichalLytek** [comment, marked as answer]
Have you installed all the deps?

**Reply by MichalLytek** [reply]
Here in package.json the `typegraphql-prisma` is missing:
https://github.com/MichalLytek/typegraphql-nestjs/issues/18

**Reply by Hasilt** [reply]
The problem solved after installing `typegraphql-prisma`. I was following instructions on `typegraphql-nestjs` repo
